app/services/loader.js

```diff
@@ -7,7 +7,7 @@ import httpStatus from 'npm:http-status';
 import objectToFormData from 'npm:object-to-formdata';
 import fetch from 'fetch';
 import { clone, assign, merge } from 'lodash';
-const bodyAllowedIn = ['POST', 'PUT'];
+const bodyAllowedIn = ['PATCH', 'POST', 'PUT'];
```

Comment:
@schedutron Why do we need this?

Reply:
Because otherwise `PATCH` is treated as a method for which body is not allowed, which results in the patch body being treated as a query string (you can verify that in Firefox's network requests section).